koku/api/provider/serializers.py

```diff
@@ -202,7 +202,7 @@ class ProviderSerializer(serializers.ModelSerializer):
     """Serializer for the Provider model."""
 
     uuid = serializers.UUIDField(allow_null=True, required=False)
-    name = serializers.CharField(max_length=256, required=True, allow_null=False, allow_blank=False)
+    name = serializers.CharField(required=True, allow_null=False, allow_blank=False)
```

Comment:
`max_length` is required to avoid DDoS attack (according to a report we got from the security team in Red Hat).